File: C++ programming/PROBLEMS  ON LOOPING/Program13.cpp
#include<iostream>
using  namespace std;
void Display(int iNo)
{
    int iCnt  = 0;
    for(iCnt = iNo ; iCnt >=1; iCnt--)
    {
        cout<<iCnt<<endl;
    }
}
int main()
{
int iValue = 0 ;
cout<<"Enter the number : \n"<<endl;
cin>>iValue;
Display(iValue);

    return 0;
}

File: JAVA Programming/PROBLEM ON LOOPING/Program13.java
import java.util.*;
class Program12
{
    public static void Display(int iValue)
    {
        int iCnt  = 0;
        for(iCnt = iValue; iCnt >= 1; iCnt--)
        {
            System.out.println(iCnt);
        }
    }
    public static void main(String Args[])
    {
        Scanner sobj  = new Scanner(System.in);

        System.out.println("Enter the Number : ");
        int iNo1  = sobj.nextInt();

        Display(iNo1);
    }
}